### PR TITLE
Add human-readable functionality to the internal transactions list and small display enhancements

### LINKED ISSUE #93: Improve display of internal transactions on /tx
## Overview
It would improve user experience if the internal transactions in the transaction view got the same UI treatment as the Logs tab.

## Acceptance criteria
- internal transactions, eg. [here](https://www.teloscan.io/tx/0xcfb64bca2f55829f899a181e5b878f2cb5b43cd315087e5d8b664fbd43bc67c4#internal), should have the same appearance and view mode toggle as the Logs tab

## Technical considerations
N/A

### LINKED ISSUE #222: Add wei toggle to ERC20 transfer list values on transaction pages
# Description

Add a toggle on the transaction page in the ERC20 transfer list to show the total transfer value in wei, like we do for the transaction value itself already.

# Acceptance

User can toggle between currency view & wei view.

### PR DESCRIPTION
**Pull request by tomtomcrypto**
# Fixes #93 #173 #215 #218 #220 #221 #222 #228 #229 #204

## Description

Adding human-readable functionality to the internal transactions list and small display enhancements.

## Tasks

- [x] #93 
- [x] #173
- [x] #204
- [x] #215
- [x] #218
- [x] #222 
- [x] #221 
- [x] #220 
- [x] #228
- [x] #229
- [x] Add related contract for logs & internal transactions

## Test scenarios

- [x] Users can navigate to the internal transaction tab of a transaction and toggle between human-readable mode or normal mode.
- [x] Users can navigate to transaction general tab and toggle wei amount on the erc20 transaction list if there are any
- [x] Users can navigate to transaction general tab and see how much the gas fee was in USD
- [x] Users can navigate to transaction internal transaction or log tab and see the related contract of each log / transaction

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/components/Transaction/InternalTxnsTable.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        parsedItxs: {
+            type: Array,
+            required: true,
+        },
+    },
+    data () {
+        return {
+
+        }
+    },
```
Comment (donnyquixotic) [inline src/components/Transaction/InternalTxnsTable.vue:22]: remove empty `data()`

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

**Comment by tomtomcrypto** [comment]
Tracking a bug this introduced on some logs

**Comment by tomtomcrypto** [comment]
> > Tracking a bug this introduced on some logs, ie: https://deploy-preview-227--dev-mainnet-teloscan.netlify.app/tx/0x49780341cf3f7b71d85e02dfbc7d58933044a53e0e96dbc018297faf1b903301#eventlog
> 
> Fixed
> 
> Checking why order of internal transactions is messed up sometimes on navigation, should be good to review after that

Fixed in latest commit, good to review

**Comment by EJSG** [comment]
Damn dude this PR is amazing, everything looks really good. Only comment is that the h3 for No logs / internal transactions looks kind of silly (really large) because our typography isnt set up yet. I suggest switching them to just use a `<p>`

e.g. https://deploy-preview-227--teloscan-stage.netlify.app/tx/0x80f0051d8782bb0be772eba5f0ca1a7bcbe84c4266e1b4e7f59b6cac5c5a2747#eventlog
and https://deploy-preview-227--dev-mainnet-teloscan.netlify.app/tx/0xe525d969b79e88c6147135a07737dfe6eaad74e6c967bd595aaa8d93ce201d03#eventlog
image

**Review by EJSG — CHANGES_REQUESTED** [review]
just the one small comment. seriously amazing work killing so many issues with this

**Comment by tomtomcrypto** [comment]
> just the one small comment. seriously amazing work killing so many issues with this

Yeah it is the same problem on the logs page actually, never got around to it, I'll change it.

**Comment by tomtomcrypto** [comment]
Used an H5, size seems good and it is easier to align vertically

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**